howfairis/readme.py

```diff
@@ -6,13 +6,29 @@
 
 
 class Readme:
+    """Container for the README of a repository
+
+    Args:
+        filename: Name of README
+        text: Content of README
+        file_format: Format of README. It is used to ignore commented out badges.
+        ignore_commented_badges: If False commented out badges will be considered.
+
+    Attributes:
+        filename (str): Name of README
+        text (str): Content of README
+        file_format (ReadmeFormat): Format of README
```

Comment:
None can be passed in constructor
```suggestion
        filename (str, None): Name of README
        text (str, None): Content of README
        file_format (ReadmeFormat, None): Format of README
```